test/test_inference/test_imports.py

```diff
@@ -101,6 +101,16 @@ def test_correct_zip_package_behavior(Script, inference_state, environment, code
     assert value.py__package__() == []
 
 
+@pytest.mark.parametrize("code,name", [
+    ("from pkg.", "nested"),
+    ("from pkg.nested.", "nested_module")
+])
+def test_zip_package_import_complete(Script, environment, code, name):
+    sys_path = environment.get_sys_path() + [str(pkg_zip_path)]
+    completions = Script(code, project=Project('.', sys_path=sys_path)).complete()
+    assert name in [c.name for c in completions]
```

Comment:
Can we maybe assert all names here? like `assert names == [c.name for c in completions]`?